base/db/R/get.machine.info.R

```diff
@@ -0,0 +1,83 @@
+#' Get machine information from db
+#' @param host host information
+#' @param input.args input args for existing records
+#' @param input.id input id for existing records
+#' @param con database connection
+#'
+#' @return list of machine, input, and dbfile records
+#' @author Betsy Cowdery, Michael Dietze, Ankur Desai, Tony Gardella, Luke Dramko
+
+get.machine.info <- function(host, input.args, input.id = NULL, con = NULL) {
```

Comment:
Might as well use underscore instead of dot in the name here to avoid possible confusion with S3 methods. These days I only use dot where it's needed for consistency with existing code, and the db package is already inconsistent about it 😉 
```suggestion
get_machine_info <- function(host, input.args, input.id = NULL, con = NULL) {
```

Reply:
Need to change the usages below to match, naturally